cron.yaml

```diff
@@ -0,0 +1,6 @@
+cron: 
+- description: "Removes inneligible domains"
+  url: "/api/v2/remove-ineligible-domains"
+  schedule: every monday 09:00
+  timezone: america/new_york
```

Comment:
To answer the question about deploying this: can you add a change to the Makefile to run `gcloud app deploy cron.yaml` as part of the deploy step (similar to what's done for app.yaml)?

Reply:
Done